eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -111,6 +105,21 @@ ms.service: $service
   return "$header`n$ReadmeContent"
 }
 
+function RetrieveCodeOwners($targetDirectory) {
+  & "$ToolPath/retrieve-code-owners" --target-directory "/sdk/${{ parameters.ServiceDirectory }}/" `
+    --root-directory "$targetDirectory" `
+    --vso-owning-users "CodeOwners"
+  
+  $author = "ramya-rao-a"
+  $msauthor = "ramyar"
+  $docCodeOwner = "$env:CODEOWNERS"
```

Comment:
Does this actually work? I don't think the CODEOWNER env variable is processed until after DevOps process the variable setting. 

If we plan to consume the tool in this way perhaps we should have it simply output the value to the console or perhaps have the data written to a file that can be consumed by our PS scripts.

Reply:
We could even consider adding the necessary classes in our library to make it a PS module that we could load.

Reply:
Right now, the native way of reading code owners is to read from *.json file in `Update-DocsMsMetadata.ps1`.

I prefer to add PSModule, so that we do not need to read from console or file

Reply:
While a PSModule might be somewhat interesting it will be a little more difficult to author and get published and consumed.

I agree that parsing the output isn't perhaps the preferred way it might be the most straight forward way and is ultimately not much different then setting the devops variable as DevOps just parses that from the output as well.

Reply:
Whatever option you come up with I'd like to make sure it works for the cpp use case as well.